app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/filters/dateFilter.tsx

```
@@ -137,6 +157,40 @@ export function DateFilter({
     onSelect(range?.from.toISOString() ?? null, range?.to.toISOString() ?? null);
   };
 
+  // Add keyboard shortcut support
+  useEffect(() => {
+    if (!isOpen) return;
+
+    const handleKeyPress = (e: KeyboardEvent) => {
+      const key = e.key.toUpperCase();
+      const preset = DATE_PRESETS.find((p) => p.shortcut === key);
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)